3rd-party.txt

```diff
@@ -3262,7 +3262,7 @@ pygit2 - libgit2 bindings in Python
 
 ----------------------------------------------------------------------
 
-      LINKING EXCEPTION
+			LINKING EXCEPTION
```

Comment:
can you change tab to space?

Reply:
@jihyeonyi fixed on https://github.com/openvinotoolkit/datumaro/commit/f35d656aa41e3490dc652864540b4ebee5635b20